go.sh

```diff
@@ -26,29 +26,35 @@ if [ "${LOCATIONS_CSV}" == "" ]; then
     LOCATIONS_CSV=/
 fi
 
+NAXSI_LISTEN_PORT="10418"
```

Comment:
this lgtm, renaming it to that could be useful, we were planning to use http for sysdig anyway so that seems fine